ext/hal/ti/simplelink/source/ti/devices/cc32xx/inc/hw_gpio.h

```diff
@@ -465,7 +465,7 @@
 #define GPIO_O_GPIO_PCTL        0x0000052C  // This register is not used in
                                             // cc3xx. equivalant register exsist
                                             // outside GPIO IP (refer
-                                            // PAD*_config register in the
```

Comment:
I don't think this file needs to be changed. The comments are referring to some register name, not Zephyr function name.

Reply:
Automatic name conversion bug, fixed.

Reply:
LGTM